src/style.css

```diff
@@ -1,3 +1,132 @@
 body {
   margin: 0;
+  display: block;
+  padding: 0;
+  box-sizing: border-box;
+
+  overflow-x: hidden;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.logo_image {
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.nav {
+  display: block;
+}
+
+.moyo_menu {
+  display: flex;
+  list-style-type: none;
+}
+
+.menu-item {
+  margin-right: 30px;
+}
+
+.menu-item:first-child {
+  margin-left: 0;
+}
+
+.menu-item:last-child {
+  margin-left: 0;
+  padding-right: 50px;
+}
+
+.menu_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to include fallback fonts for `.menu_link` (line 62), in case the primary font isn't available. This ensures that your text remains readable across different platforms and devices.